overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-early-reboot/fips-reboot.conf

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=CoreOS Early Reboot
+# We need to convert /run/ignition-modified-kargs into a triggering
```

Comment:
Hmm, maybe instead the flag file should just be called something more agnostic upstream. E.g. `/run/coreos-reboot-required` or something. Then we can just have it touch the same file and avoid this drop-in.

Ahh, except `rdcore kargs --create-if-changed` uses `O_EXCL` so we'd have to run the FIPS code *after* `ignition-kargs.service` and not before.

Reply:
The other thing we could do is just make it a triggering condition upstream, with a single trigger.

Reply:
https://github.com/coreos/fedora-coreos-config/pull/1135

Reply:
https://github.com/coreos/fedora-coreos-config/pull/1135

Reply:
WFM though I still wonder whether we really need multiple triggering conditions at all. Why not simplify further and have a single "reboot required" trigger? Avoids having to ship new dracut modules just to add drop-ins entirely.

Reply:
Okay, I can make that change.  How would you feel about dropping the O_EXCL so we're not managing that ordering dependency indefinitely?

Reply:
Yeah, makes sense. I can take care of that.

Reply:
Went ahead and created https://github.com/coreos/coreos-installer/pull/577 so I'd have something to link to.

Reply:
The flag file change was https://github.com/coreos/fedora-coreos-config/pull/1136, for the record.